extensions/smallrye-reactive-messaging-rabbitmq/deployment/src/main/java/io/quarkus/smallrye/reactivemessaging/rabbitmq/deployment/RabbitMQBuildTimeConfig.java

```diff
@@ -12,4 +14,21 @@
      */
     @ConfigItem
     public RabbitMQDevServicesBuildTimeConfig devservices;
+
+    /**
+     * The credentials provider name.
+     */
+    @ConfigItem
+    public Optional<String> credentialsProvider = Optional.empty();
+
+    /**
+     * The credentials provider bean name.
+     * <p>
+     * It is the {@code &#64;Named} value of the credentials provider bean. It is used to discriminate if multiple
```

Comment:
It should use @Identified and not @Named.

Reply:
I think, in Quarkus, this is supposed to be `@Named`.  Look at `DataSourceRuntimeConfig` in the `quarkus-datasource` extension. It uses `@Named` only with no support for `@Identifier`

Reply:
@mkouba what do you recommend? @Named or @Identifier. Isn't it another abuse of @Named?

Reply:
Just to be clear, currently this _has_ to be `@Named` unless we update the `quarkus-credentials` extension and any extension that uses its `CredenitlasProviderFinder`; and that's a lot.

Reply:
+1 for `io.smallrye.common.annotation.Identifier` or another custom qualifier.

Reply:
I'm all for this change but isn't this something that should be do in a separate PR Again, the use of `@Named` is dictated by the `credentials` extension. For the change to be meaningful the `datasource`, `oidc`, `db2-reactive`, `mssql-reactive`, `mysql-reactive`, and `postgres-reactive` need to be updated as well.

Reply:
And we still need to support the `@Named` annotation as well while it's only deprecated.

Reply:
@kdubb agreed. Let's be consistent and make the change in one go in another PR if we think it has value. But this needs to be discussed separately.